### DISCUSSION TITLE
Drop out, slow stream counter, help needed

### DISCUSSION THREAD

**Discussion by Kiwi915** [schreibfaul1/ESP32-audioI2S#393] [Q&A]
Hi,
Has anybody a idea how to make a drop out, slow stream counter ?
I want to count , how often the message comes :slow stream, drop outs are possible.
The counter was not the problem, but how to get the event, which show the slow stream text , in a variable to count ?

Kind regards Michaela

**Comment by Zinfandel22** [comment]
Can't you look for the value in info when == slow steam and add a counter+=1?

void audio_info(const char *info) {
  Serial.print("info        ");
  Serial.println(info);
}
